components/Cards/examples/EditReorderCollectionViewController.swift

```diff
@@ -34,11 +34,11 @@ class EditReorderCollectionViewController: UIViewController,
   var containerScheme: MDCContainerScheming
 
   var colorScheme: MDCColorScheming {
```

Comment:
Seems like these two computed vars can be removed entirely and their use replaced with `containerScheme.colorScheme`.

Reply:
These have been updated.